Frontend/src/Components/Graph/CustomGraph.js

```diff
@@ -150,6 +169,10 @@ function getOptions(now, secondsRetained) {
  * @returns
  */
 export default function CustomGraph(props) {
+  const { colorMode } = useColorMode();
+
+  const borderCol = colorMode === "light" ? colors.light.border : colors.dark.border;
```

Comment:
There's a lot of these terinary statments throughout the codebase - I think we should have one source of truth for dark vs light mode colors - and a function that controls that - we can pass a key and it'll return the correct color

`borderCol = getColor(border);`
```
function getColor(key)
{
    let colorMode = howeverWeGetColorMode();
    return colors['colorMode'][key]
```

Reply:
I also passed colorMode into this function since a hook is used to get colorMode. I tried setting color mode up in the theme so that there would be no need for a funciton/handling color mode on a per-component basis, but it didn't work.